microcosm_flask/swagger/api.py

```diff
@@ -0,0 +1,38 @@
+"""
+API interfaces for swagger operations.
+
+"""
+from typing import Mapping
+
+from marshmallow import Schema
+from marshmallow.fields import Field
+
+from microcosm_flask.swagger.parameters.base import ParameterBuilder
+from microcosm_flask.swagger.schema import iter_fields
+
+
+def build_schema(schema: Schema) -> Mapping:
+    """
+    Build JSON schema from a marshmallow schema.
+
+    """
+    fields = list(iter_fields(schema))
```

Comment:
It will eventually make sense to have a `SchemaBuilder` to mirror the `ParameterBuilder` structure. We don't yet have the same need for flexibility, but it'd make the code easier to follow.